config/routes.rb

```diff
@@ -1,8 +1,30 @@
+# Rails.application.routes.draw do
+#   devise_for :users, ActiveAdmin::Devise.config
+#   ActiveAdmin.routes(self)
+#   resources :books, only: %I[index show]
+#   resources :rents, only: %I[index create]
+#   resources :book_suggestions, only: %I[create]
+#   mount_devise_token_auth_for 'User', at: 'auth'
+#   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+#   require 'sidekiq/web'
+#   mount Sidekiq::Web => '/sidekiq'
+# end
+
 Rails.application.routes.draw do
+  devise_for :users, ActiveAdmin::Devise.config
```

Comment:
I prefer to get another model to admin_users, because users contains a lot of more fields, and admin users (usually) only got an email and a password.
It seems to me, to execute `rails g active_admin:install User`
Instead of `rails g active_admin:install`

No problem, this comment it's only a suggestion, it's not necessary to change it now :wink:

Reply:
In this way, you permit to every user can access information in the active admin. Usually, we use pundit to assign roles and permits